layouts/partials/footer.html

```diff
@@ -96,3 +96,34 @@
 
 </script>
 {{- end }}
+
+{{- if (not .Site.Params.disableCodeCopy) }}
+<script>
+    document.querySelectorAll('pre > code').forEach((codeblock) => {
+        const container = codeblock.parentNode.parentNode;
+
+        const copybutton = document.createElement('div');
```

Comment:
this could be done to hide `$1`
```diff
- const copybutton = document.createElement('div');
+ const copybutton = document.createElement('button');

Reply:
It doesn't make the text underlined on hover like `$1` does, but I'll make the change anyways since it is really a button.

Reply:
@kdkasad Yes, it's a button we don't need it to be underlined

Reply:
Oh right! I hadn't made the style changes to turn it into a button at that point. I'll fix that up.